python/ray/_private/memory_monitor.py

```diff
@@ -127,12 +127,7 @@ def __init__(self, error_threshold=0.95, check_interval=1):
     def get_memory_usage(self):
         psutil_mem = psutil.virtual_memory()
         total_gb = psutil_mem.total / (1024**3)
-        used_gb = total_gb - psutil_mem.available / (1024**3)
-
-        # Linux, BSD has cached memory, which should
-        # also be considered as unused memory
-        if hasattr(psutil_mem, "cached"):
-            used_gb -= psutil_mem.cached / (1024**3)
+        used_gb = psutil_mem.used / (1024**3)
```

Comment:
Btw, note; 

According to the doc; https://psutil.readthedocs.io/en/latest/

used: memory used, calculated differently depending on the platform and designed for informational purposes only. total - free does not necessarily match used.

available: the memory that can be given instantly to processes without the system going into swap. This is calculated by summing different memory values depending on the platform and it is supposed to be used to monitor actual memory usage in a cross platform fashion.